data/xml/1995.iwpt.xml

```diff
@@ -0,0 +1,287 @@
+    </paper>
+    <paper id="26">
+      <title>A Corpus-based Probabilistic Grammar with Only Two Non-terminals</title>
+      <author><first>Satoshi</first><last>SEKINE</last></author>
+      <author><first>Ralph</first><last>GRISHMAN</last></author>
+      <pages>216-223</pages>
+      <url>1995.iwpt-1.26</url>
+      <abstract>The availability of large, syntactically-bracketed corpora such as the Penn Tree Bank affords us the opportunity to automatically build or train broad-coverage grammars, and in particular to train probabilistic grammars. A number of recent parsing experiments have also indicated that grammars whose production probabilities are dependent on the context can be more effective than context-free grammars in selecting a correct parse. To make maximal use of context, we have automatically constructed, from the Penn Tree Bank version 2, a grammar in which the symbols S and NP are the only real nonterminals, and the other non-terminals or grammatical nodes are in effect embedded into the right-hand-sides of the S and NP rules. For example, one of the rules extracted from the tree bank would be S -&gt; NP VBX JJ CC VBX NP [1] ( where NP is a non-terminal and the other symbols are terminals – part-of-speech tags of the Tree Bank). The most common structure in the Tree Bank associated with this expansion is (S NP (VP (VP VBX (ADJ JJ) CC (VP VBX NP)))) [2]. So if our parser uses rule [1] in parsing a sentence, it will generate structure [2] for the corresponding part of the sentence. Using 94% of the Penn Tree Bank for training, we extracted 32,296 distinct rules ( 23,386 for S, and 8,910 for NP). We also built a smaller version of the grammar based on higher frequency patterns for use as a back-up when the larger grammar is unable to produce a parse due to memory limitation. We applied this parser to 1,989 Wall Street Journal sentences (separate from the training set and with no limit on sentence length). Of the parsed sentences (1,899), the percentage of no-crossing sentences is 33.9%, and Parseval recall and precision are 73.43% and 72 .61%.</abstract>
+    </paper>
+    <paper id="27">
+      <title>HEURISTICS AND PARSE RANKING</title>
+      <author><first>B.</first><last>Srinivas</last></author>
+      <author><first>Christine</first><last>Doran</last></author>
```

Comment:
Is this [Srinivas Bangalore](https://www.aclweb.org/anthology/people/s/srinivas-bangalore/)? It seems the name is backwards, though I guess it's in the variants file.

Reply:
it is